null_safety_examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -213,9 +207,9 @@ void miscDeclAnalyzedButNotTested() {
 
   (Map<Chest, Treasure> _opened) {
     // #docregion arrow-long
-    Treasure openChest(Chest chest, Point where) =>
+    Treasure? openChest(Chest chest, Point where) =>
         _opened.containsKey(chest) ? null : _opened[chest] = Treasure(where)
-          ..addAll(chest.contents);
+          ?..addAll(chest.contents);
```

Comment:
This looks like it was a bug before. I think the intention was probably

```dart
Treasure? openChest(Chest chest, Point where) => _opened.containsKey(chest)
    ? null
    : _opened[chest] = (Treasure(where)..addAll(chest.contents));
```

Right?